### ISSUE TITLE
okhttp to retrofit and some rxJava?

### ISSUE THREAD

**Issue by gugakatsi** [Swati4star/Travel-Mate#9]

**Comment by Swati4star** [comment]
@gugakatsi  Can you list some advantage of why should I use retrofit, instead of okhttp?

**Comment by gugakatsi** [comment]
@Swati4star of course, so first of all maybe i am wrong here so that's why i wrote ? at the end :D so don't judge me ,  without a doubt retrofit code i guess cleaner and much easier to read right? also it's really handsome while you're decomposing app for example, also it is great with alongside with rxJava, i don't know this about okhttp and also retrofit also uses okhttp in some situations when it's needed,  so i think it's kinda more modern now , :D that was all i could say

**Comment by Swati4star** [comment]
I know, Retrofit code is cleaner, and easier to read. If you come across any other benefit too, mention it here.
I thought it would be a lot of effort to do this task. If you would like to work on moving the code from okhttp to retrofit. you are welcome. :)

**Comment by gugakatsi** [comment]
@Swati4star  yeap i would work on it , i have exams right now , so i will start working in few days

**Comment by Swati4star** [comment]
Great. Just post a message over here when you will start working on it.

**Comment by Kartik2016** [comment]
Is the issue still open ?

**Comment by gugakatsi** [comment]
Yeap, we gonna do it after refactoring app to mvp , in end of September
probably

On Aug 24, 2017 5:22 PM, "Kartik2016" <notifications@github.com> wrote:

Is the issue still open ?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Swati4star/Travel-Mate/issues/9#issuecomment-324633323>,
or mute the thread

**Comment by Kartik2016** [comment]
Is there a way i could contribute too?

**Comment by gugakatsi** [comment]
Sure , check issue #10 and write on which module you gonna work

**Comment by Sumeet-Rukeja** [comment]
Greetings of the day!
I want to contribute to this project and i think i'll be helpful in shifting the project from okhttp to retrofit

**Comment by gugakatsi** [comment]
@Sumeet-Rukeja sorry for late response , i'll write down tomorrow evening .

**Comment by Sumeet-Rukeja** [comment]
Hi, any updates on this issue??

**Comment by gugakatsi** [comment]
@Sumeet-Rukeja , make indepedent package data and inside that remote package , start with destinations module , refactor those okHttp to RetroFit . tell me what you think about that.

**Comment by Sumeet-Rukeja** [comment]
Ok, i'll start with Destinations module and will use retrofit 2.0+.

**Comment by Sumeet-Rukeja** [comment]
Hi @gugakatsi @Swati4star, i have completed integrating retrofit in Destinations module and have created a pull request . Please, verify it and let me know about the next module to continue this process.
